spacy/ml/extract_spans.py

```diff
@@ -57,9 +57,9 @@ def _get_span_indices(ops, spans: Ragged, lengths: Ints1d) -> Ints1d:
     for i, length in enumerate(lengths):
         spans_i = spans[i].dataXd + offset
         for j in range(spans_i.shape[0]):
-            indices.append(ops.xp.arange(spans_i[j, 0], spans_i[j, 1]))  # type: ignore[call-overload, index]
+            indices += list(range(spans_i[j, 0], spans_i[j, 1]))  # type: ignore
```

Comment:
I think you'd need to ignore the types here, but I haven't checked:

```suggestion
            indices.extend(range(spans_i[j, 0], spans_i[j, 1]))
```